suite/tests/client-interface/stolen-reg.c

```diff
@@ -223,17 +262,21 @@ main(int argc, char **argv)
 
     join_thread(thread);
 
+#ifndef RISCV64
```

Comment:
Please make it clear this is only temporarily disabled: add something like `TODO i#3544: Add synchall support to RISC-V`